restcomm/restcomm.mgcp/src/main/java/org/restcomm/connect/mgcp/MockMediaGateway.java

```diff
@@ -400,6 +439,26 @@ private void modifyConnection(final Object message, final ActorRef sender) {
     private void deleteConnection(final Object message, final ActorRef sender) {
         final ActorRef self = self();
         final DeleteConnection dlcx = (DeleteConnection) message;
+        if (dlcx.getConnectionIdentifier() == null) {
+            connEndpointMap.values().removeAll(Collections.singleton(dlcx.getEndpointIdentifier().getLocalEndpointName()));
```

Comment:
can we log the result of this operation, i think it could be worth to print it, whether we deleted something or not...

Reply:
@jaimecasero you mean the operation to remove from the Map? Yes I can add this at the log statement later